crates/subspace-runtime/src/lib.rs

```diff
@@ -74,7 +74,7 @@ use subspace_core_primitives::{
     Randomness, RecordsRoot, RootBlock, SegmentIndex, SolutionRange, PIECE_SIZE,
 };
 use subspace_runtime_primitives::{
-    opaque, AccountId, Balance, BlockNumber, Hash, Index, Moment, Signature, CONFIRMATION_DEPTH_K,
```

Comment:
If you removed usage here, then go ahead and remove the constant from runtime primitives too or else it may still be referenced from somewhere and it'll be problematic.